src/server/pachyderm_test.go

```diff
@@ -1191,6 +1191,63 @@ func TestPipelineFailure(t *testing.T) {
 	require.True(t, strings.Contains(jobInfo.Reason, "datum"))
 }
 
+func TestEgressFailure(t *testing.T) {
+	// setup
```

Comment:
nit, I don't think comments like this are useful and only take up space. Comments that describe something that isn't obvious from the code is helpful, but I don't think the comments in this test tell me anything that couldn't be easily gleaned from the code.